src/Components/DisplayAllArticles.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import axios from "axios";
+import ArticleTemplate from "./ArticleTemplate";
+
+class DisplayAllArticles extends Component {
+  constructor() {
+    super();
+    this.state = {
+      allArticles: []
+    };
+  }
+
+  componentDidMount() {
+    this.getAllArticles();
+  }
+
+  async getAllArticles() {
+    axios.get("http://localhost:3000/api/v0/articles")
+    .then(response => {
+      this.setState({
+        allArticles: response.data
+      });
+    });
+  }
```

Comment:
I would suggest you use `await` and a try-catch block

```
try {
  // code to run on success
} catch() {
  // code to run on failure
}
```

Reply:
Refactored with async/await and try/catch.